changelog/10482.bugfix.md

```diff
@@ -0,0 +1,2 @@
+Fixes #10471 - Cannot Override LexicalSyntacticFeaturizer in 2.8.x
+Adds a new optional config argument `prefix_suffix_case_sensitive` which can be set to `False` to ignore cases in prefix and suffix functions.
```

Comment:
```suggestion
Adds a new optional config argument `prefix_suffix_case_sensitive` which can be set to `False` to ignore cases in prefix and suffix functions.
```
The `Allows LexicalSyntacticFeaturizer to be overridden.` part should go into a separate change log file `10471.bugfix.md`. Also, this file (`10482.bugfix.md`) should be `10482.improvement.md`.

Reply:
Separated

Reply:
@JEM-Mosig done.